include/api.php

```diff
@@ -1485,6 +1485,31 @@ function api_users_search($type)
 /// @TODO move to top of file or somewhere better
 api_register_func('api/users/search', 'api_users_search');
 
+/**
+ * Return user objects
+ *
+ * @see https://developer.twitter.com/en/docs/accounts-and-users/follow-search-get-users/api-reference/get-users-lookup
+ *
+ * @param string $type Return format: json or xml
+ *
+ * @return array|string
+ * @throws UnauthorizedException
+ */
+function api_users_lookup($type)
+{
+	$users = array();
+	foreach (explode(',', $_REQUEST['user_id']) as $id) {
```

Comment:
Could you check if the request value is set and throw an exception if not?

Reply:
Twitter does not return an error when `user_id` is missing. It does however return a 404 error when the results are empty. I think we should do that too.

Reply:
I would still check for the existence of the value just to avoid a PHP Notice if it's missing.